src/scripts.js

```diff
@@ -14,38 +14,89 @@ import domUpdates from './domUpdates';
 
 // ** Button Query Selectors **
 const darkLightModeButton = document.querySelector('.dark-light-mode-js');
+const homeButton = document.querySelector('.go-home-js');
 const currentTripsButton = document.querySelector('.go-to-current-js');
 const plannedTripsButton = document.querySelector('.go-to-planned-js');
 const pastTripsButton = document.querySelector('.go-to-past-js')
-const homeButton = document.querySelector('.go-home-js')
-const nextDestinationButton = document.querySelector('.next-destination-js');
+const makeBookingButton = document.querySelector('.make-booking-js');
 
 // ** Display Query Selectors **
+const homeDisplay = document.querySelector('.dashboard-js');
+const currentDisplay = document.querySelector('.detailed-current-trips-js');
+const plannedDisplay = document.querySelector('.detailed-planned-trips-js');
+const pastDisplay = document.querySelector('.detailed-past-trips-js');
+const bookingDisplay = document.querySelector('.booking-js');
 
-
+// ** Event Listeners **
 darkLightModeButton.addEventListener('click', toggleDarkMode);
+homeButton.addEventListener('click', showHome);
+currentTripsButton.addEventListener('click', showCurrentDetails);
+plannedTripsButton.addEventListener('click', showPlannedDetails);
+pastTripsButton.addEventListener('click', showPastDetails);
+makeBookingButton.addEventListener('click', showBookingDetails);
+
 
-let traveler;
-let travelers = [];
-let trips = [];
-let destinations = [];
+window.traveler;
+window.travelers = [];
+window.trips = [];
+window.destinations = [];
 
 window.onload = Promise.all([apiCalls.getTravelersData(), apiCalls.getTripsData(), apiCalls.getDestinationsData()])
   .then(data => {
     data[0].travelers.forEach(traveler => travelers.push(traveler))
     data[1].trips.forEach(trip => trips.push(trip))
     data[2].destinations.forEach(destination => destinations.push(destination))
-    instantiation();
+    getData();
   })
 
-function instantiation() {
-  traveler = new Traveler(travelers[7]);
-  domUpdates.greetTraveler(traveler);
-  domUpdates.updateTravelerStats(traveler, trips, destinations);
+function getData() {
+  window.traveler = new Traveler(travelers[7]);
+  domUpdates.greetTraveler(window.traveler);
+  domUpdates.updateTravelerStats(window.traveler, window.trips, window.destinations);
+}
+
+function showHome() {
+  domUpdates.manageClassList('remove', 'hidden', homeDisplay);
+  domUpdates.manageClassList('remove', 'hidden', makeBookingButton);
+  hideDetailedDisplays();
+}
+
+function showCurrentDetails() {
+  domUpdates.manageClassList('add', 'hidden', homeDisplay);
```

Comment:
Could a 'toggle' help here?

Reply:
I had a lot of heartache managing toggle with this, which I found disappointing. I think part of it had to do with having a lot of moving pieces around at the same time? Either way, I would love to use that rather than add & remove. But they do get the job done.